sklearn/feature_selection/_rfe.py

```diff
@@ -457,6 +458,24 @@ class RFECV(RFE):
         ``grid_scores_[i]`` corresponds to
         the CV score of the i-th subset of features.
 
+        .. deprecated:: 0.23
+        The `grid_scores_` attribute is deprecated in version 0.23 in favor
+        of `cv_results_` and will be removed in version 0.25
+
+    cv_results_ : dict of floats
+        A dict with keys:
+
+        split(i)_score : float
+        corresponds to the CV score of the i-th subset of features
+
+        mean_score : float
+        mean of split(i)_score values in dict
+
+        std_score : float
+        std of split(i)_score values in dict
```

Comment:
Maybe the dictionary keys could be listed as a bullet list?

Reply:
I am okay with this for now, we use this formatting for other places where we return dicts (`fetch_openml`).